vtrace/platforms/linux.py

```diff
@@ -24,6 +24,8 @@
 
 from ctypes import *
 import ctypes.util as cutil
+from vtrace.platforms.posix import PT_CONTINUE
+from ptrace.binding.func import PTRACE_EVENT_EXEC, PTRACE_O_TRACEEXEC
```

Comment:
Lets get these defined here in the linux module.  Avoiding external dependancies. ( other than that, i love this change )